tests/cases/compiler/moduleResolutionWithSymlinks_preserveSymlinks.ts

```diff
@@ -0,0 +1,24 @@
+// @noImplicitReferences: true
+
+// @traceResolution: true
+// @preserveSymlinks: true
+// @moduleResolution: node
+
+// @filename: /linked/index.d.ts
+// @symlink: /app/node_modules/linked/index.d.ts,/app/node_modules/linked2/index.d.ts
+export { real } from "real";
```

Comment:
Would it make sense to write a unit test (possible replacing this)? We seem to test similar behavior in `src/harness/unittests/moduleResolution.ts`